.travis.yml

```diff
@@ -13,6 +13,10 @@ notifications:
   email: false
 script:
   - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-  - julia -e 'Pkg.clone(pwd()); Pkg.build("Mocking"); Pkg.test("Mocking"; coverage=true)';
+  - |
+    julia -e '
+      VERSION >= v"0.7.0-DEV.3656" && using Pkg
+      VERSION >= v"0.7.0-DEV.5183" ? Pkg.instantiate() : Pkg.clone(pwd())
+      Pkg.build("Mocking"); Pkg.test("Mocking"; coverage=true)'
 after_success:
   - julia -e 'cd(Pkg.dir("Mocking")); Pkg.add("Coverage"); using Coverage; Codecov.submit(process_folder())'
```

Comment:
What should I do address the `Pkg.dir` deprecation here?

Reply:
Something like
```julia
VERSION >= v"0.7.0-DEV.3656" && using Pkg
Pkg.add("Coverage")
using Coverage
dir = if VERSION >= v"0.7.0-DEV.5183"
    dirname(Base.find_package("Mocking"))
else
    Pkg.dir("Mocking")
end
cd(dir)
Codecov.submit(process_folder())
```
should work, but OH MAN is the ugly.

Reply:
Aren't we standing in the repo when doing the `after_success`? So just remove the `Pkg.dir` completely?